packages/dashboard/src/components/widgets/list.tsx

```diff
@@ -20,6 +21,38 @@ export type WidgetsProps = {
   messageOverrides: DashboardMessages;
 };
 
+export const isQueryWidget = (widget: DashboardWidget): widget is LineScatterChartWidget =>
+  widget.properties.queryConfig != null;
+
+const getWidgetsWithHiddenPropertiesRemoved = (dashboardConfiguration: {
```

Comment:
can we move this into `customization/widgets/lineScatterChart/component` since it will only work with that widget?

Reply:
+1

Reply:
done